### PR TITLE
Added fish support

### PR DESCRIPTION
**Pull request by max2000warlord**

### CODE REVIEW

**Comment by Sharktheone** [comment]
Thanks for the PR! I need to look at this later, when I can properly test it.

File: src/install_libs.py [hunk 1]
```diff
@@ -151,10 +153,19 @@ def add_lib_path(self):
                     return
 
         path = get_rc_path()
-        
         path = os.path.expanduser(path)
+        shell = os.getenv("SHELL")
```
Comment (Sharktheone) [inline src/install_libs.py:157]: Could you please initialize the `command` variable here, so LSPs/linters don't complain?
Essentially, just like that:

```suggestion
        shell = os.getenv("SHELL")
        
        command = None
        
```

**Review by Sharktheone — CHANGES_REQUESTED** [review]
One simple thing, rest looks good!

**Comment by Sharktheone** [comment]
Can you also check the signing of your commits? It shows up as `unverified`

File: src/install_libs.py (hunk 1 above)
Comment (max2000warlord) [reply src/install_libs.py:157]: done

Comment (max2000warlord) [reply src/install_libs.py:157]: Done

**Review by Sharktheone — APPROVED** [review]
Okay, looks good now!

**Comment by Sharktheone** [comment]
Thanks for the PR!

**Pull request merged**